### PR TITLE
login

### PR DESCRIPTION
**Pull request by phemmylintry**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -106,7 +136,6 @@
     'django.contrib.messages.middleware.MessageMiddleware',
 ]
 
-CORS_ORIGIN_ALLOW_ALL = True
```
Comment (KeelTech) [inline config/settings/base.py:109]: put this back!

File: keel/authentication/models.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def create_superuser(self, email, password, **extra_fields):
 
 
 class User(AbstractBaseUser, PermissionsMixin):
-    username = None
+    username = models.CharField(max_length=255, null=True)
```
Comment (KeelTech) [inline keel/authentication/models.py:46]: revert!

File: keel/authentication/models.py [hunk 3]
```diff
@@ -83,6 +83,9 @@ class TimeStampedModel(models.Model):
     class Meta:
         abstract = True
 
+class CustomToken(TimeStampedModel):
+    token = models.CharField(max_length=255, blank=False, null=True, default=None)
+
```
Comment (KeelTech) [inline keel/authentication/models.py:88]: add user_id

File: config/settings/base.py [hunk 4]
```diff
@@ -216,3 +239,6 @@
 API_BASE_URL = env('API_BASE_URL')
 # CONN_MAX_AGE=600
 
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+USER_SECRET_KEY = ["secret"]
```
Comment (KeelTech) [inline config/settings/base.py:244]: will this be an array

Comment (phemmylintry) [reply config/settings/base.py:249]: TBH, I don't know lol, let me just remove it.,

File: config/settings/base.py (hunk 1 above)
Comment (phemmylintry) [reply config/settings/base.py:109]: done

**Review by KeelTech — APPROVED** [review]

**Pull request merged**